src/config/DataCommons.ts

```diff
@@ -27,7 +27,7 @@ export const DataCommons: DataCommon[] = [
         fileType: "pdf",
         prefix: "CDS_",
         downloadPrefix: "CDS_",
-        fileTransferManifestName: "CDS_Data_Loading_Template-file-manifest",
+        fileTransferManifestName: "CDS_Data_Loading_Template_File_Manifest",
```

Comment:
My assumption with the bug ticket CRDCDH-1360 would be that we want to be completely consistent with the rest of the node TSV names. If that's the case, the correct values would be:

|Data Common|fileTransferManifestName|Note|
|:-|:-|:-|
|CDS| `CDS_Data_Loading_Template_file`|
|CTDC|`CTDC_Data_Loading_Template_data_file`|CTDC uses "data_file" instead
|ICDC|`ICDC_Data_Loading_Template_file `|

Any thoughts on this? I might be misinterpreting the bug